drivers/lm75/include/lm75_regs.h

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2021 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_lm75
+ *
+ * @{
+ * @file
+ * @brief       Registers for the lm75 and derived (lm75a and tmp1075) temperature sensors.
+ *
+ * @author      Vitor Batista <vitor.batista@ml-pa.com>
+ *
+ * @}
+ */
+
+#ifndef LM75_REGS_H
+#define LM75_REGS_H
+
+
+/* LM75 register list */
+#define LM75_TEMP_REG            (0x00)	/* Temperature register */
+#define LM75_CONF_REG            (0x01)	/* Configuration register */
+#define LM75_THYST_REG           (0x02)	/* Hysteresis register */
+#define LM75_TOS_REG             (0x03)	/* Overtemperature shutdown register */
+
+/* Device Operation mode */
+#define NORMAL_MODE			0
+#define SHUTDOWN_MODE		1
+
+/* Device Thermostat  operation mode */
+#define COMPARATOR_MODE	    0
+#define INTERRUPT_MODE	    1
+
+/* OS polarity */
+#define OS_ACTIVE_LOW		0
+#define OS_ACTIVE_HIGH		1
```

Comment:
```suggestion
#define OS_ACTIVE_LOW         0
#define OS_ACTIVE_HIGH        1
```

Coding style mandates spaces, not tabs